src/bouwdossiers/tests/data/WABO_centrum.xml

```diff
@@ -11,7 +11,7 @@
     <locaties>
       <locatie>
         <straatnaam>Lauriergracht</straatnaam>
-        <huisnummer>116</huisnummer>
+        <huisnummer>1,161</huisnummer>
```

Comment:
Sanity check: why are comma's used as digit group separators here? All the data seems to be in nl_NL format, so if thousand-separators have to be used shouldn't that be a "period" (`.`)? And why is a separator used at all?

Reply:
Not my choice, but something which came out of the source systems, which are outside of our control.